src/apps/amp/ethereum/amp.farm.contract-position-fetcher.ts

```diff
@@ -45,14 +38,39 @@ export class EthereumAmpFarmContractPositionFetcher extends SingleStakingFarmTem
     key: (address: string) => `apps-v3:balance:ethereum:amp:api-data:${address}`,
```

Comment:
@Cache won't be necessary because the BE is already caching addresses

Reply:
Got it.